Classes/Settings/SettingsViewController.swift

```diff
@@ -203,7 +203,7 @@ GitHubSessionListener {
 
     private func onSignOut() {
         let title = NSLocalizedString("Are you sure?", comment: "")
-        let message = NSLocalizedString("All of your accounts will be signed out, and their bookmarks will be removed. Do you want to continue?", comment: "")
+        let message = NSLocalizedString("You will be signed out from all of your accounts. Do you want to sign out?", comment: "")
         let alert = UIAlertController.configured(title: title, message: message, preferredStyle: .alert)
```

Comment:
Should this be an action sheet instead, like the delete-photo action sheet in Photos?

Reply:
🤔 that does fit the app style and will look better when we add the (sign out, sign out of all,
Cancel) once we are able to do selective sign out

Reply:
@Huddie should I prepare this part for an ActionSheet style control?

Reply:
Ya, I think ActionSheet makes sense. Fits the style and works better with the future selective sign-out option.

Reply:
@Huddie I've changed it!
![image](https://user-images.githubusercontent.com/1848357/49632680-ea1f7000-f9f6-11e8-94fa-eb7ee8b33663.png)
What do you think? It was just an idea to change the primary action string to `sign out of all`

Reply:
I’d leave it as “Sign out” for now

Reply:
@Huddie I've reverted the change